repository/src/main/java/org/alfresco/service/cmr/search/CategoryService.java

```diff
@@ -122,7 +122,7 @@ public enum Depth {IMMEDIATE, ANY};
      * @param sortByName boolean
      */
     @Auditable(parameters = {"storeRef", "aspectName", "pagingRequest", "sortByName"})
-    PagingResults<ChildAssociationRef> getRootCategories(StoreRef storeRef, QName aspectName, PagingRequest pagingRequest, boolean sortByName);
+    Collection<ChildAssociationRef> getRootCategories(StoreRef storeRef, QName aspectName, PagingRequest pagingRequest, boolean sortByName);

```

Comment:
As I commented for `TaggingService` - I would strongly prefer adding methods than changing their signatures.